docs/api/geo_datasets.csv

```diff
@@ -12,6 +12,7 @@ Dataset,Type,Source,Size (px),Resolution (m)
 `GBIF`_,Points,Citizen Scientists,-,-
 `GlobBiomass`_,Masks,Landsat,"45,000x45,000",100
 `iNaturalist`_,Points,Citizen Scientists,-,-
+`LandCover.ai Geo`_,"Imagery, Masks",Aerial,-,0.25--0.5
```

Comment:
Can you add the size of each image (in pixels)? Can be a range if it varies by a lot